test/puppetlabs/puppetdb/scf/migrate_test.clj

```diff
@@ -57,11 +57,11 @@
 
         (testing "should attempt a partial migration if there are migrations missing"
           (clear-db-for-testing!)
-          ;; We are using migration 13 here because it is isolated enough to be able
+          ;; We are using migration 28 here because it is isolated enough to be able
```

Comment:
Umm, why are we modifying an existing test, and using it for our migration test? Shouldn't we go and create a new one?